src/zimcheck/checks.cpp

```diff
@@ -244,3 +244,27 @@ void test_articles(const zim::Archive& archive, ErrorLogger& reporter, ProgressB
         }
     }
 }
+
+void test_redirect_loop(const zim::Archive& archive, ErrorLogger& reporter) {    
+    std::cout << "[INFO] Checking for redirect loops..." << std::endl;
+
+    int chained_redirection_limit = 21;
```

Comment:
In libzim, the limit is 50 (https://github.com/openzim/libzim/blob/master/src/entry.cpp#L75).
No value is better than the other but it would be nice to have the same value.

Reply:
This `watchdog` limit shoudl be made available to be reusable IMO.

Reply:
I'm not sure we want the watchdog limit being part of our public libzim API.

Reply:
@mgautierfr How should we secure otherwise that the zimcheck limit is the same as the one in the libzim?

Reply:
We don't have to secure it. There is no problem if the limit is not the same. Just one tool more tolerant than the other.
It is nice to have the same value but this is not mandatory.
Make the watchdog public is price to high for a nice coherence between two different checks.